package.json

```diff
@@ -24,7 +24,7 @@
   "license": "MIT",
   "dependencies": {
     "loopback-connector": "^2.2.0",
-    "mongodb": "^2.0.35",
+    "mongodb": "2.1.22-alpha4",
```

Comment:
Any version number string containing `-<suffix>` is a pre-release version accordig to semver, see http://semver.org/#spec-item-9:

> A pre-release version indicates that the version is unstable and might not satisfy the intended compatibility requirements as denoted by its associated normal version. 

I don't think it's a good idea to depend on an unstable version in a stable release of our connector.

I am proposing to use something like `~2.1.21` instead